cluster/deployment/scratchnete/config.yaml

```diff
@@ -6,6 +6,7 @@ infra:
 pulumiProjectConfig:
   default:
     cloudSql:
+      enabled: true
```

Comment:
@nicu-da probably not gonna merge this but wanna check the config before I deploy?

Reply:
good catch switched